src/main/java/co/omise/Client.java

```diff
@@ -95,11 +95,6 @@ public Client(Requester requester) {
      * @throws ClientException if client configuration fails (e.g. when TLSv1.2 is not supported)
      */
     protected OkHttpClient buildHttpClient(Config config) throws ClientException {
-        CertificatePinner.Builder pinner = new CertificatePinner.Builder();
```

Comment:
Just wonder, the code never use `pinner`..
So it is not pinned at the beginner

Reply:
We were removed the certificate pinning since we were the issue about certificate I think it already a year ago. So, this PR to wipe it all certificate pinning code from the project. 🙂